replica/table.cc

```diff
@@ -1137,7 +1137,9 @@ compaction_group::update_main_sstable_list_on_compaction_completion(sstables::co
 future<>
 table::compact_all_sstables() {
     co_await flush();
-    co_await _compaction_manager.perform_major_compaction(as_table_state());
+    co_await for_each_compaction_group([this] (compaction_group& cg) {
+        return _compaction_manager.perform_major_compaction(cg.as_table_state());
```

Comment:
In parallel, compaction_manager will serialize as needed (and show the rest as pending).